dev-app/src/screens/CollectCardPaymentScreen.tsx

```diff
@@ -121,12 +129,16 @@ export default function CollectCardPaymentScreen() {
     if (enableInterac) {
       paymentMethods.push('interac_present');
     }
+    const routingPriority = {
+      requested_priority: inputValues.requestPriority,
+    };
     const paymentMethodOptions = {
       card_present: {
         request_extended_authorization:
           inputValues.requestExtendedAuthorization,
         request_incremental_authorization_support:
           inputValues.requestIncrementalAuthorizationSupport,
+        routing: routingPriority,
```

Comment:
this is currently not passed to SDK's createPaymentIntent method call for non Internet readers. Should update L193 on this file to include this option